src/docs/constants/docsItems.ts

```diff
@@ -34,6 +34,7 @@ export const docsItems: DocItems = [
         .map((componentType) => ({
           href: `/docs/components/${componentType.type}`,
           label: componentType.title,
+          description: `${componentType.title}-${componentType.subtitle}`,
```

Comment:
This should be fine in being just the subtitle. If you do want to combine both properties can we include a whitespaces in between so they read as: "Badges - Highlight key information with attention-grabbing badges".

Reply:
I agree, IMO just the subtitle would work the best (if I'm not mistaken, it is used only once right now in generate metadata and the label is used there too, so having the title in both label and description is redundant id say)